src/checkout/api_default.go

```diff
@@ -126,8 +126,8 @@ func (a Checkout) PaymentMethodsBalance(req *CheckoutBalanceCheckRequest, ctxs .
 }
 
 /*
-PostPaymentSession Creates a payment session.
-Provides the data object that can be used to start the Checkout SDK. To set up the payment, pass its amount, currency, and other required parameters. We use this to optimise the payment flow and perform better risk assessment of the transaction.  For more information, refer to [How it works](https://docs.adyen.com/checkout#howitworks).
+PaymentSession Creates a payment session.
+Provides the data object that can be used to start the Checkout SDK. To set up the payment, pass its amount, currency, and other required parameters. We use this to optimise the payment flow and perform better risk assessment of the transaction.  For more information, refer to [How it works](https://docs.adyen.com/online-payments#howitworks).
```

Comment:
I see docs did redirect the `/checkout` to `/online-payments` but the anchor is not present in the new link. (the same link occurs 3 more times in this PR)

```suggestion
Provides the data object that can be used to start the Checkout SDK. To set up the payment, pass its amount, currency, and other required parameters. We use this to optimise the payment flow and perform better risk assessment of the transaction.  For more information, refer to [How it works](https://docs.adyen.com/online-payments).
```